src/static/helpers/slasHelper.ts

```diff
@@ -126,12 +123,12 @@ export async function authorize(
       response_type: 'code',
       ...(parameters.usid && {usid: parameters.usid}),
     },
-    fetchOptions,
+    redirect,
   };
 
   const response = await slasClient.authorizeCustomer(options, true);
 
-  const redirectUrl = response.url || response.headers?.get('location');
+  const redirectUrl = response.headers?.get('location') || response.url;
```

Comment:
Why the switch?

Reply:
Is this maybe the only change that needs to be made is this one line?